### ISSUE TITLE
Add stylelint-a11y

### ISSUE THREAD

**Issue by spaceninja** [spaceninja/stylelint-config-spaceninja#10]
Review stylelint-a11y to see if it should be included: https://github.com/YozhikM/stylelint-a11y
